packages/dialect-react-ui/components/Notifications/screens/NewSettings/Telegram.tsx

```diff
@@ -81,8 +86,7 @@ const Telegram = () => {
   const saveTelegram = async () => {
     try {
       const value = telegramUsername.replace('@', '');
-      const address = await createAddress({ value });
-      await toggleSubscription({ enabled: true, address });
+      await createAddress({ value });
```

Comment:
Could you please clarify this change please? Why are we no longer toggling telegram subscription after submitting?

Reply:
The deleted line is a hack. Previously we needed to also create subscription before verifying the address

Reply:
I'm just thinking: after creating telegram account for the first time, subscription should be enabled, leads to better UX(no need to additionally switch the toggle), wdyt?

Reply:
This makes sense, but I think if we do this, we need to apply this to all channels, wdym? Since rn you need to toggle manually everywhere except telegram

Reply:
@zhelezkov @fsher

Reply:
yeah it's a good point. wanna cover it?

Reply:
@zhelezkov will do in this PR

Reply:
@zhelezkov done